define language targets and charsets for the bootloader theme.

© Niall Walsh <niallwalsh@users.berlios.de>
Signed-off-by: Stefan Lippers-Hollmann <s.l-h@gmx.de>

--- a/themes/openSUSE/data-boot/languages
+++ b/themes/openSUSE/data-boot/languages
@@ -1 +1,16 @@
+cs_CZ
+da_DK
+de_DE
+en_GB
 en_US
+es_ES
+fr_FR
+it_IT
+ja_JP
+nl_NL
+pl_PL
+pt_BR
+pt_PT
+ro_RO
+ru_RU
+uk_UA
--- a/themes/openSUSE/src/dia_lang.inc
+++ b/themes/openSUSE/src/dia_lang.inc
@@ -19,71 +19,72 @@
 /.la.name   3 def
 
 /lang.defaultnames [
-  [ "af_ZA" "us"                   false "Afrikaans" ]						% Afrikaans
+% [ "af_ZA" "us"                   false "Afrikaans" ]						% Afrikaans
 % [ "ar_EG" "arabic"               true  "\u064a\u0628\u0631\u0639" ]				% Arabic
-  [ "ar_EG" "arabic"               true  "\ufef2\ufe91\ufeae\ufecb" ]				% Arabic
-  [ "ast_ES" "es"                  false "Asturianu" ]						% Asturian
-  [ "bg_BG" "us"                   false "Български" ]						% Bulgarian
-  [ "bn_BD" "us"                   false "\u09ac\u09be\u0982\u09b2\u09be" ]			% Bengali
-  [ "bs_BA" "us"                   false "Bosanski" ]						% Bosnian
-  [ "ca_ES" "es"                   false "Català" ]						% Catalan
-  [ "cs_CZ" "cz-us-qwertz"         false "Čeština" ]						% Czech
-  [ "cy_GB" "us"                   false "Cymraeg" ]						% Welsh
+% [ "ar_EG" "arabic"               true  "\ufef2\ufe91\ufeae\ufecb" ]				% Arabic
+% [ "ast_ES" "es"                  false "Asturianu" ]						% Asturian
+% [ "bg_BG" "us"                   false "Български" ]						% Bulgarian
+% [ "bn_BD" "us"                   false "\u09ac\u09be\u0982\u09b2\u09be" ]			% Bengali
+% [ "bs_BA" "us"                   false "Bosanski" ]						% Bosnian
+% [ "ca_ES" "es"                   false "Català" ]						% Catalan
+  [ "cs_CZ" "cz"                   false "Čeština" ]						% Czech
+% [ "cy_GB" "us"                   false "Cymraeg" ]						% Welsh
   [ "da_DK" "dk"                   false "Dansk" ]						% Danish
-  [ "de_DE" "de-latin1-nodeadkeys" false "Deutsch" ]						% German
-  [ "el_GR" "gr"                   false "Ελληνικά" ]						% Greek
-  [ "en_GB" "uk"                   false "English (UK)" ]					% English
+  [ "de_DE" "de" false "Deutsch" ]						% German
+% [ "el_GR" "gr"                   false "Ελληνικά" ]						% Greek
+  [ "en_AU" "us"                   false "English (AU)" ]					% English
+  [ "en_GB" "gb"                   false "English (UK)" ]					% English
   [ "en_US" "us"                   false "English (US)" ]					% English
   [ "es_ES" "es"                   false "Español" ]						% Spanish
-  [ "et_EE" "et"                   false "Eesti" ]						% Estonian
-  [ "fa_IR" "us"                   true "\ufbfd\ufeb4\u0631\ufe8e\ufed3" ]			% Farsi
-  [ "fi_FI" "fi-latin1"            false "Suomi" ]						% Finnish
-  [ "fr_FR" "fr-latin1"            false "Français" ]						% French
-  [ "gl_ES" "es"                   false "Galego" ]						% Galician
-  [ "ka_GE" "us"                   false "ქართული" ]						% Kartuli
-  [ "gu_IN" "us"                   false "\u0a97\u0ac1\u0a9c\u0ab0\u0abe\u0aa4\u0ac0" ]		% Gujarati
-  [ "he_IL" "us"                   false "תירבע" ]						% Hebrew
+% [ "et_EE" "et"                   false "Eesti" ]						% Estonian
+% [ "fa_IR" "us"                   true "\ufbfd\ufeb4\u0631\ufe8e\ufed3" ]			% Farsi
+% [ "fi_FI" "fi-latin1"            false "Suomi" ]						% Finnish
+  [ "fr_FR" "fr"            false "Français" ]						% French
+% [ "gl_ES" "es"                   false "Galego" ]						% Galician
+% [ "ka_GE" "us"                   false "ქართული" ]						% Kartuli
+% [ "gu_IN" "us"                   false "\u0a97\u0ac1\u0a9c\u0ab0\u0abe\u0aa4\u0ac0" ]		% Gujarati
+% [ "he_IL" "us"                   false "תירבע" ]						% Hebrew
 % [ "hi_IN" "us"                   false "\u0939\u093f\u0928\u094d\u0926\u0940" ]		% Hindi
-  [ "hi_IN" "us"                   false "\u0939\u093f\u0902\u0926\u0940" ]			% Hindi
-  [ "hr_HR" "croat"                false "Hrvatski" ]						% Croatian
-  [ "hu_HU" "hu"                   false "Magyar" ]						% Hungarian
-  [ "id_ID" "us"                   false "Bahasa Indonesia" ]					% Indonesian
+% [ "hi_IN" "us"                   false "\u0939\u093f\u0902\u0926\u0940" ]			% Hindi
+% [ "hr_HR" "hr"                false "Hrvatski" ]						% Croatian
+% [ "hu_HU" "hu"                   false "Magyar" ]						% Hungarian
+% [ "id_ID" "us"                   false "Bahasa Indonesia" ]					% Indonesian
   [ "it_IT" "it"                   false "Italiano" ]						% Italian
-  [ "ja_JP" "jp106"                false "日本語" ]						% Japanese
-  [ "jv_ID" "us"                   false "Basa Jawa" ]						% Javanese
+  [ "ja_JP" "jp"                false "日本語" ]						% Japanese
+% [ "jv_ID" "us"                   false "Basa Jawa" ]						% Javanese
 % [ "km_KH" "khmer"                false "\u1781\u17d2\u1798\u17c2\u179a" ]			% Khmer
-  [ "km_KH" "khmer"                false "Khmer" ]						% Khmer
-  [ "ko_KR" "korean"               false "한글" ]						% Korean
-  [ "ky_KG" "us"                   false "Кыргызча" ]						% Kirghiz
-  [ "lo_LA" "us"                   false "\u0e9e\u0eb2\u0eaa\u0eb2\u0ea5\u0eb2\u0ea7" ]		% Lao
-  [ "lt_LT" "lt.baltic"            false "Lietuvių" ]						% Lithuanian
-  [ "mk_MK" "us"                   false "Македонски" ]						% Macedonian
-  [ "mr_IN" "us"                   false "\u092e\u0930\u093e\u0920\u0940" ]			% Marathi
-  [ "nb_NO" "no-latin1"            false "Norsk" ]						% Norwegian
+% [ "km_KH" "khmer"                false "Khmer" ]						% Khmer
+% [ "ko_KR" "korean"               false "한글" ]						% Korean
+% [ "ky_KG" "us"                   false "Кыргызча" ]						% Kirghiz
+% [ "lo_LA" "us"                   false "\u0e9e\u0eb2\u0eaa\u0eb2\u0ea5\u0eb2\u0ea7" ]		% Lao
+% [ "lt_LT" "lt.baltic"            false "Lietuvių" ]						% Lithuanian
+% [ "mk_MK" "us"                   false "Македонски" ]						% Macedonian
+% [ "mr_IN" "us"                   false "\u092e\u0930\u093e\u0920\u0940" ]			% Marathi
+% [ "nb_NO" "no-latin1"            false "Norsk" ]						% Norwegian
   [ "nl_NL" "us"                   false "Nederlands" ]						% Dutch
-  [ "nn_NO" "no-latin1"            false "Nynorsk" ]						% Nynorsk
-  [ "pa_IN" "us"                   false "\u0a2a\u0a70\u0a1c\u0a3e\u0a2c\u0a40" ]		% Punjabi
-  [ "pl_PL" "Pl02"                 false "Polski" ]						% Polish
-  [ "pt_BR" "br-abnt2"             false "Português (Brasil)" ]					% Portuguese (Brazilian)
-  [ "pt_PT" "pt-latin1"            false "Português" ]						% Portuguese
+% [ "nn_NO" "no"            false "Nynorsk" ]						% Nynorsk
+% [ "pa_IN" "us"                   false "\u0a2a\u0a70\u0a1c\u0a3e\u0a2c\u0a40" ]		% Punjabi
+  [ "pl_PL" "pl"                 false "Polski" ]						% Polish
+  [ "pt_BR" "br"             false "Português (Brasil)" ]					% Portuguese (Brazilian)
+  [ "pt_PT" "pt"            false "Português" ]						% Portuguese
   [ "ro_RO" "us"                   false "Română" ]						% Romanian
-  [ "ru_RU" "ruwin_alt-UTF-8"      false "Русский" ]						% Russian
-  [ "si_LK" "us"                   false "\u0dc3\u0dd2\u0d82\u0dc4\u0dbd" ]			% Sinhala
-  [ "sk_SK" "sk-qwertz"            false "Slovenčina" ]						% Slovak
-  [ "sl_SI" "slovene"              false "Slovenščina" ]					% Slovenian
-  [ "sr_RS" "sr-cy"                false "Srpski" ]						% Serbian
-  [ "sv_SE" "sv-latin1"            false "Svenska" ]						% Swedish
-  [ "ta_IN" "us"                   false "\u0ba4\u0bae\u0bbf\u0bb4\u0bcd" ]			% Tamil
-  [ "tg_TJ" "tj"                   false "тоҷикӣ" ]						% Tadjik
-  [ "th_TH" "us"                   false "\u0e20\u0e32\u0e29\u0e32\u0e44\u0e17\u0e22" ] 	% Thai
-  [ "tr_TR" "trq"                  false "Türkçe" ]						% Turkish
+  [ "ru_RU" "ru"      false "Русский" ]						% Russian
+% [ "si_LK" "us"                   false "\u0dc3\u0dd2\u0d82\u0dc4\u0dbd" ]			% Sinhala
+% [ "sk_SK" "sk-qwertz"            false "Slovenčina" ]						% Slovak
+% [ "sl_SI" "slovene"              false "Slovenščina" ]					% Slovenian
+% [ "sr_RS" "sr-cy"                false "Srpski" ]						% Serbian
+% [ "sv_SE" "sv-latin1"            false "Svenska" ]						% Swedish
+% [ "ta_IN" "us"                   false "\u0ba4\u0bae\u0bbf\u0bb4\u0bcd" ]			% Tamil
+% [ "tg_TJ" "tj"                   false "тоҷикӣ" ]						% Tadjik
+% [ "th_TH" "us"                   false "\u0e20\u0e32\u0e29\u0e32\u0e44\u0e17\u0e22" ] 	% Thai
+% [ "tr_TR" "trq"                  false "Türkçe" ]						% Turkish
   [ "uk_UA" "ua"                   false "Українська" ]						% Ukrainian
-  [ "vi_VN" "us"                   false "Tiếng Việt" ]						% Vietnamese
-  [ "wa_BE" "be-latin1"            false "Walon" ]						% Walloon
-  [ "xh_ZA" "us"                   false "isiXhosa" ]						% Xhosa
-  [ "zh_CN" "us"                   false "简体中文" ]						% Simplified Chinese
-  [ "zh_TW" "us"                   false "繁體中文 (台灣)" ]					% Traditional Chinese
-  [ "zu_ZA" "us"                   false "isiZulu" ]						% Zulu
+% [ "vi_VN" "us"                   false "Tiếng Việt" ]						% Vietnamese
+% [ "wa_BE" "be-latin1"            false "Walon" ]						% Walloon
+% [ "xh_ZA" "us"                   false "isiXhosa" ]						% Xhosa
+% [ "zh_CN" "us"                   false "简体中文" ]						% Simplified Chinese
+% [ "zh_TW" "us"                   false "繁體中文 (台灣)" ]					% Traditional Chinese
+% [ "zu_ZA" "us"                   false "isiZulu" ]						% Zulu
 ] def
 
 
--- a/themes/openSUSE/data-install/languages
+++ b/themes/openSUSE/data-install/languages
@@ -1,62 +1,16 @@
-af_ZA
-ar_EG
-ast_ES
-bn_BD
-bs_BA
-bg_BG
-ca_ES
 cs_CZ
-cy_GB
 da_DK
 de_DE
-et_EE
 en_GB
 en_US
 es_ES
-fa_IR
 fr_FR
-gl_ES
-ka_GE
-gu_IN
-el_GR
-hi_IN
-id_ID
-hr_HR
 it_IT
-he_IL
 ja_JP
-jv_ID
-km_KH
-ko_KR
-ky_KG
-lo_LA
-lt_LT
-mr_IN
-hu_HU
-mk_MK
 nl_NL
-nb_NO
-nn_NO
 pl_PL
-pt_PT
 pt_BR
-pa_IN
+pt_PT
 ro_RO
 ru_RU
-zh_CN
-si_LK
-sk_SK
-sl_SI
-sr_RS
-fi_FI
-sv_SE
-tg_TJ
-ta_IN
-th_TH
-vi_VN
-zh_TW
-tr_TR
 uk_UA
-wa_BE
-xh_ZA
-zu_ZA
--- a/themes/openSUSE/src/keytables.inc
+++ b/themes/openSUSE/src/keytables.inc
@@ -54,46 +54,46 @@
 
 % array must not be empty
 /keymaps [
-  [ "Arabic"                  "arabic"               .undef                      ]
-  [ "Belgian"                 "be-latin1"            keymap.be-latin1            ]
-  [ "Canadian (Multilingual)" "cn-latin1"            keymap.cn-latin1            ]
-  [ "Croatian"                "croat"                keymap.croat                ]
-  [ "Czech"                   "cz-us-qwertz"         keymap.cz-us-qwertz         ]
-  [ "Czech (qwerty)"          "cz-lat2-us"           keymap.cz-lat2-us           ]
+%  [ "Arabic"                  "arabic"               .undef                      ]
+  [ "Belgian"                 "be"                   keymap.be-latin1            ]
+  [ "Canadian (Multilingual)" "ca"                   keymap.cn-latin1            ]
+  [ "Croatian"                "hr"                   keymap.croat                ]
+  [ "Czech"                   "cz"                   keymap.cz-us-qwertz         ]
   [ "Danish"                  "dk"                   keymap.dk                   ]
   [ "Dutch"                   "nl"                   keymap.nl                   ]
   [ "Dvorak"                  "dvorak"               keymap.dvorak               ]
-  [ "English (UK)"            "uk"                   keymap.uk                   ]
+  [ "English (IE)"            "ie,gb"                keymap.uk                   ]
+  [ "English (UK)"            "gb"                   keymap.uk                   ]
   [ "English (US)"            "us"                   .undef                      ]
   [ "Estonian"                "et"                   keymap.et                   ]
-  [ "Finnish"                 "fi-latin1"            keymap.fi-latin1            ]
-  [ "French"                  "fr-latin1"            keymap.fr-latin1            ]
+  [ "Finnish"                 "fi"                   keymap.fi-latin1            ]
+  [ "French"                  "fr"                   keymap.fr-latin1            ]
   [ "French (Canada)"         "cf"                   keymap.cf                   ]
-  [ "French (Switzerland)"    "fr_CH-latin1"         keymap.fr_CH-latin1         ]
-  [ "German"                  "de-latin1-nodeadkeys" keymap.de-latin1-nodeadkeys ]
-  [ "German (Switzerland)"    "sg-latin1"            keymap.sg-latin1            ]
+%  [ "French (Switzerland)"    "fr=ch"                keymap.fr_CH-latin1         ]
+  [ "German"                  "de"                   keymap.de-latin1-nodeadkeys ]
+%  [ "German (Switzerland)"    "de=ch"                keymap.sg-latin1            ]
   [ "Greek"                   "gr"                   keymap.gr                   ]
   [ "Hungarian"               "hu"                   keymap.hu                   ]
-  [ "Icelandic"               "is-latin1"            keymap.is-latin1            ]
+  [ "Icelandic"               "is"                   keymap.is-latin1            ]
   [ "Italian"                 "it"                   keymap.it                   ]
-  [ "Japanese"                "jp106"                keymap.jp106                ]
-  [ "Khmer"                   "khmer"                .undef                      ]
-  [ "Korean"                  "korean"               .undef                      ]
-  [ "Lithuanian"              "lt.baltic"            keymap.lt.baltic            ]
-  [ "Norwegian"               "no-latin1"            keymap.no-latin1            ]
-  [ "Polish"                  "Pl02"                 keymap.Pl02                 ]
-  [ "Portuguese"              "pt-latin1"            keymap.pt-latin1            ]
-  [ "Portuguese (Brazil)"     "br-abnt2"             keymap.br-abnt2             ]
-  [ "Russian"                 "ruwin_alt-UTF-8"      keymap.ruwin_alt-UTF-8      ]
-  [ "Serbian"                 "sr-cy"                keymap.sr-cy                ]
-  [ "Slovak"                  "sk-qwertz"            keymap.sk-qwertz            ]
-  [ "Slovak (qwerty)"         "sk-qwerty"            keymap.sk-qwerty            ]
-  [ "Slovene"                 "slovene"              keymap.slovene              ]
+  [ "Japanese"                "jp"                   keymap.jp106                ]
+%  [ "Khmer"                   "khmer"                .undef                      ]
+  [ "Korean"                  "kr"                   .undef                      ]
+  [ "Lithuanian"              "lt"                   keymap.lt.baltic            ]
+  [ "Norwegian"               "no"                   keymap.no-latin1            ]
+  [ "Polish"                  "pl"                   keymap.Pl02                 ]
+  [ "Portuguese (PT)"         "pt"                   keymap.pt-latin1            ]
+  [ "Portuguese (BR)"         "br"                   keymap.br-abnt2             ]
+  [ "Russian"                 "ru"                   keymap.ruwin_alt-UTF-8      ]
+  [ "Serbian"                 "sr"                   keymap.sr-cy                ]
+  [ "Slovak"                  "sk"                   keymap.sk-qwertz            ]
+%  [ "Slovak (qwerty)"         "sk=qwerty"            keymap.sk-qwerty            ]
+  [ "Slovene"                 "si"                   keymap.slovene              ]
   [ "Spanish"                 "es"                   keymap.es                   ]
-  [ "Spanish (Latin America)" "la-latin1"            keymap.la-latin1            ]
-  [ "Swedish"                 "sv-latin1"            keymap.sv-latin1            ]
+  [ "Spanish (Latin America)" "latem"                keymap.la-latin1            ]
+  [ "Swedish"                 "se"                   keymap.sv-latin1            ]
   [ "Tajik"                   "tj"                   keymap.tj                   ]
-  [ "Turkish"                 "trq"                  keymap.trq                  ]
+  [ "Turkish"                 "tr"                   keymap.trq                  ]
   [ "Ukrainian"               "ua"                   keymap.ua                   ]
 ] def
 
